devolutions-gateway/src/recording/muxer.rs

```diff
@@ -0,0 +1,43 @@
+use std::{
+    path::{Path, PathBuf},
+    sync::Arc,
+    time::Duration,
+};
+
+use anyhow::Context;
+
+pub async fn remux(input: impl AsRef<Path>) -> anyhow::Result<()> {
+    let path = Arc::new(input.as_ref().to_owned());
+    // Could potentially be a very slow operation
+    tokio::task::spawn_blocking(|| remux_impl(path)).await??;
+    Ok(())
+}
+
+fn remux_impl(input: Arc<PathBuf>) -> anyhow::Result<()> {
+    std::thread::sleep(Duration::from_secs(1));
```

Comment:
question: What is the reason for this sleep? If we absolutely need to wait for one second, use an async sleep from tokio outside of the blocking thread. A thread pool is used under the hoods, so we should avoid this kind of pattern when possible.

Reply:
Sorry, this was for debugging!